content/articles/spring-webflux/index.md

```diff
@@ -0,0 +1,315 @@
+Reactive programming is a programming paradigm that supports an asynchronous, event-driven, and non-blocking approach to data processing. The reactive programming paradigm organizes events and data as streams.
+
+In the reactive programming paradigm, a request is made, other tasks are executed while waiting for the results. When the data is available, a notification is sent with the data through the callback function. The reactive programming paradigm is suitable for data-driven applications such as chat applications.
+
+In this tutorial, we are going to create a student management system using Spring webflux and MongoDB.
+
+
+### Prerequisites
+- [JDK](https://www.oracle.com/java/technologies/javase-downloads.html) installed on your computer.
+- Your favorite IDE or editor installed.
+- Knowledge of [Java](https://www.tutorialspoint.com/java/index.htm) and [Spring Boot](https://spring.io/projects/spring-boot).
+- Knowledge of [Mongodb](https://www.mongodb.com/).
+
+### Streams API
+Software developers at Netflix, Twitter, Pivotal, and Redhat created the streams API. Streams API defines four interfaces below.
+
+**Publisher**
```

Comment:
Change all the subheaders like this and further (i.e. Publisher, Subscriber, Subscription, Processor, Mono, Flux). To be formatted like this:

> \#\#\#\# Publisher

This just looks nicer and more clearly defines the subheader.